docs/translations.rst

```diff
@@ -0,0 +1,310 @@
+
+******************
+Translation Guide
+******************
+
+.. contents::
+
+You can help to translate the Read the Docs Sphinx Theme.
+
+Languages available:
+
+- Spanish contributed by Leonardo J. Caballero G.
```

Comment:
Let's create an authors file similar to https://github.com/rtfd/readthedocs.org/blob/master/AUTHORS.rst . please do this in another PR though.

Reply:
@Blendify @agjohnson removed this section into a new AUTHORS.rst file :+1: thanks for your feedback :nerd_face: